python/src/aiconfig/editor/server/server.py

```diff
@@ -17,9 +17,57 @@
 from aiconfig.model_parser import InferenceOptions
 
 
+logging.getLogger("werkzeug").disabled = True
+
+logging.basicConfig(format=core_utils.LOGGER_FMT)
+LOGGER = logging.getLogger(__name__)
+
+log_handler = logging.FileHandler("editor_flask_server.log", mode="a")
+formatter = logging.Formatter(core_utils.LOGGER_FMT)
+log_handler.setFormatter(formatter)
+
+LOGGER.addHandler(log_handler)
+
+
+class EditServerConfig(core_utils.Record):
+    server_port: int = 8080
+    aiconfig_path: Optional[str] = None
+    log_level: str | int = "INFO"
+    server_mode: str
+    parsers_module_path: Optional[str] = None
+
+
+@dataclass
+class ServerState:
+    aiconfig_runtime: AIConfigRuntime | None = None
+
+
+@dataclass(frozen=True)
+class HttpPOSTResponse:
```

Comment:
nit: rename to `HttpPostResponse` (weird that HTTP is lowercased but POST is capitalized)